.circleci/soltest.sh

```diff
@@ -55,7 +55,12 @@ get_logfile_basename() {
 }
 
 BOOST_TEST_ARGS="--color_output=no --show_progress=yes --logger=JUNIT,error,test_results/`get_logfile_basename`.xml"
-SOLTEST_ARGS="--evm-version=$EVM --evmonepath /usr/lib/libevmone.so $SOLTEST_FLAGS"
+if [ "$(uname)" == "Darwin" ]
+then
+  SOLTEST_ARGS="--evm-version=$EVM --evmonepath /usr/local/lib/libevmone.dylib $SOLTEST_FLAGS"
```

Comment:
```suggestion
  SOLTEST_ARGS="--evm-version=$EVM $SOLTEST_FLAGS"
```
If the current version runs through, I'd suggest to try using this for both Linux and MacOS, because I think it should just work for both now. Apart from that I'd say this is fine, if the CI runs confirm it works!

Reply:
yep i think the same